### PR TITLE
add deployment file

### PR DESCRIPTION
**Pull request by adebmbng**
got some errors, but I don't think the error on the server-side.
so, would you please check the code?

here is the subdomain:
https://fikomup.subagamilenia.com

### CODE REVIEW

**Comment by adebmbng** [comment]
Here is another example, the server can access the media folder
https://fikomup.subagamilenia.com/media/source/1110230045150354938524August2017.jpeg

**Comment by adebmbng** [comment]
@alfinkerin @amoschristian @dinoprojects 
please kindly check

thank you

File: .htaccess [hunk 1]
```diff
@@ -1,8 +1,6 @@
 <IfModule mod_rewrite.c>
 RewriteEngine on
 
-RewriteCond %{HTTPS} !=on
```
Comment (adebmbng) [inline .htaccess:4]: I used the gateway in my infrastructure. so we don't need any https to the service.

**Comment by amoschristian** [comment]
Approved and merged, if you can kindly re deploy it to live. Thanks

**Pull request merged**